pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -44,6 +44,18 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 		}
 	}
 
+	// When a cluster is already in progress with an upgrade we we should not start an upgrade
+	if c := resourcemerge.FindOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorProgressing); c != nil {
+		if c.Status == configv1.ConditionTrue {
+			return &precondition.Error{
+				Nested:  err,
+				Reason:  "UpgradeInProgress",
+				Message: "update target can not be changed before the current update completes",
```

Comment:
nit: "update" -> "Update".  From [the docs][1] for the very similar ClusterOperator conditions:

> All messages should start with a capital letter (like a sentence) and be written for an end user / admin to debug the problem.

[1]: https://github.com/openshift/cluster-version-operator/blame/94429fec12c7e3e709649b822cbfcf9cd55d176c/docs/dev/clusteroperator.md#L155
[1]: